File: content/en/docs/Tutorials/Movement/encoders.c
#include <webots/PositionSensor.hpp>
...

//Inside main:
PositionSensor *leftEncoders = wheel_left->getPositionSensor();     //Step 1
PositionSensor *rightEncoders = wheel_right->getPositionSensor();

leftEncoders->enable(timeStep);    //Step 2
rightEncoders->enable(timeStep);

printf("Left motor has spun %f radians", leftEncoders->getValue());     //Step 3
printf("Right motor has spun %f radians", rightEncoders->getValue());

File: content/en/docs/Tutorials/Movement/encoders.py
from controller import PositionSensor
...

leftEncoder = wheel_left.getPositionSensor()    #Step 1
rightEncoder = wheel_right.getPositionSensor()

leftEncoder.enable(timeStep)    #Step 2
rightEncoder.enable(timeStep)

print("Left motor has spun " + str(leftEncoder.getValue()) + " radians")    #Step 3
print("Right motor has spun " + str(rightEncoder.getValue()) + " radians")